cmd/package-operator-manager/components/options_test.go

```diff
@@ -49,7 +49,7 @@ func TestProvideOptions(t *testing.T) {
 						{
 							MatchExpressions: []corev1.NodeSelectorRequirement{
 								{
-									Key:      "node-role.kubernetes.io/infra",
```

Comment:
Why were the labels in this file changed?

Reply:
see above